SourceX/miniwin/misc.cpp

```diff
@@ -112,8 +112,9 @@ UINT GetDriveTypeA(LPCSTR lpRootPathName)
 
 WINBOOL DeleteFileA(LPCSTR lpFileName)

```

Comment:
Since you already changed the call site in Source, you might as well change the signature to take a string directly

Reply:
This uses `TranslateFileName`, which is used in a bunch of places

Reply:
Done